### PR TITLE
Add Ability To Configure SQLx Slow Statement Logging Settings

### PR DESCRIPTION
**Pull request by anshap1719**
## New Features

- [x] Add Ability To Configure SQLx Slow Statement Logging Settings

### CODE REVIEW

**Comment by anshap1719** [comment]
Can you please confirm if I should fix that clippy failure? The error isn't coming from my changes.

**Comment by tyt2y3** [comment]
If you wouldn't mind, that'll be great to fix clippy warnings!

**Comment by anshap1719** [comment]
@tyt2y3 I don't mind at all, the reason I asked is because they look like re-exports and hence could be in use by crate users. So might not be the best idea to do that..

**Comment by tyt2y3** [comment]
If clippy warns about unused re-export, then nothing actually have been exported.

**Comment by anshap1719** [comment]
@tyt2y3 I see. Makes sense. Will fix them then

**Comment by anshap1719** [comment]
Thank you @tyt2y3

**Comment by tyt2y3** [comment]
We fixed them on #2057

File: src/database/mod.rs [hunk 1]
```diff
@@ -147,6 +151,8 @@ impl ConnectOptions {
             max_lifetime: None,
             sqlx_logging: true,
             sqlx_logging_level: log::LevelFilter::Info,
+            sqlx_slow_statements_logging_level: log::LevelFilter::Warn,
```
Comment (tyt2y3) [inline src/database/mod.rs:154]: I think we should set the default level to `LevelFilter::Off`

File: src/driver/sqlx_mysql.rs [hunk 2]
```diff
@@ -52,6 +52,10 @@ impl SqlxMySqlConnector {
             opt = opt.disable_statement_logging();
         } else {
             opt = opt.log_statements(options.sqlx_logging_level);
+            opt = opt.log_slow_statements(
```
Comment (tyt2y3) [inline src/driver/sqlx_mysql.rs:55]: ... and only call this function if level filter is not Off.

File: src/database/mod.rs [hunk 3]
```diff
@@ -269,11 +275,31 @@ impl ConnectOptions {
         self
     }
 
+    /// Set SQLx slow statements logging level and duration threshold (default INFO, 1s)
```
[the remark this answers is not in the document]
Comment (tyt2y3) [reply src/database/mod.rs:278]: We should not enable it by default

**Comment by anshap1719** [comment]
@tyt2y3 I was wondering, should I put effort into being able to do this when user disables SQLx logging and uses sea-orm's debug-print instead? I will create a separate PR for it if it makes sense.

**Comment by tyt2y3** [comment]
umm I don't think it can easily be done. disabling sqlx logging would disable slow log.

**Comment by anshap1719** [comment]
@tyt2y3 Maybe I didn't word it clearly. I was just asking if "Slow statements logging" when `debug-print` is enabled, makes sense, completely independant of sqlx.

**Comment by tyt2y3** [comment]
I think not necessary. Because I can imagine it'd take a lot of effort.

**Comment by anshap1719** [comment]
@tyt2y3 Got it. Yup. Thanks for confirming.

**Review by tyt2y3 — APPROVED** [review]

**Comment by tyt2y3** [comment]
Thank you. I'll merge now. Feel free to suggest other improvements!

**Comment by anshap1719** [comment]
@tyt2y3 Will do. Thank you. Do you know yet when the next release will be, or are there no plans yet?

**Comment by tyt2y3** [comment]
Probably today. Or the next few days.

**Comment by anshap1719** [comment]
@tyt2y3 Perfect! Thank you.

**Pull request merged**